cmd/root.go

```diff
@@ -80,34 +64,38 @@ var rootCmd = &cobra.Command{
 			logs.Error(err)
 		}
 
-		r := mux.NewRouter()
+		e := echo.New()
+		e.Validator = dto.NewValidator()
+		e.Use(middleware.Recover())
+
+		routes.Register(e.Group("/api"))
 
-		r.PathPrefix("/media/partyhall").Handler(http.StripPrefix("/media/partyhall", http.FileServer(http.Dir(utils.GetPath("images")))))
-		r.PathPrefix("/media/karaoke").Handler(http.StripPrefix("/media/karaoke", http.FileServer(http.Dir(utils.GetPath("karaoke")))))
+		log.Info("Registered routes: ")
+		for _, r := range e.Routes() {
+			log.Info("\t- " + r.Path)
+		}
 
-		routes.Register(r.PathPrefix("/api").Subrouter())
+		logs.Infof("PartyHall app is listening on %v\n", config.GET.Web.ListeningAddr)
+		logs.Error(e.Start(config.GET.Web.ListeningAddr))
 
-		if services.WEBAPP_FS != nil {
-			r.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-				mode := "booth"
-				if utils.IsRemote(r) {
-					mode = "admin"
-				}
+		/*
+			r.PathPrefix("/media/partyhall").Handler(http.StripPrefix("/media/partyhall", http.FileServer(http.Dir(utils.GetPath("images")))))
```

Comment:
No longer needed, when it will be needed (feature view pictures from admin) it should be re-introduced in a secure manner (Can only access it as admin, on the current event only) in the photobooth module.

Photobooth module should also be decoupled from core ASAP